airbyte-integrations/connectors/source-plausible/integration_tests/sample_config.json

```diff
@@ -2,4 +2,4 @@
   "api_key": "your_api_key",
   "site_id": "airbyte.com",
   "start_date": "2022-01-01"
-}
+}
```

Comment:
Please remove the file from this PR as it is unrelated to the LinkedIn Ads source connector.